tests/acquisition/covidcast/test_signal_dash_data_generator.py

```diff
@@ -12,8 +12,7 @@
 import pandas as pd
 
 # first party
-from delphi.epidata.acquisition.covidcast import signal_dash_data_generator
-from delphi.epidata.acquisition.covidcast.signal_dash_data_generator import (
+from delphi.epidata.maintenance.signal_dash_data_generator import (
```

Comment:
should we move this file from `tests/acquisition/covidcast/` to `tests/maintenance/` to match the `src` file that got moved?

Reply:
👍